opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py

```diff
@@ -31,9 +31,13 @@
 )
 from opentelemetry._metrics.instrument import UpDownCounter as APIUpDownCounter
 from opentelemetry.sdk._metrics.measurement import Measurement
-from opentelemetry.sdk._metrics.measurement_consumer import MeasurementConsumer
 from opentelemetry.sdk.util.instrumentation import InstrumentationScope
 
+if TYPE_CHECKING:
```

Comment:
Should `InstrumentationScope` be in this if as well?

Reply:
Not necessary, this `TYPE_CHECKING` is there to avoid a problem that happens because of a circular import, only `MeasurementConsumer` causes that error.